test/test_regression_cli.py

```diff
@@ -44,40 +59,43 @@ def _test_factory(test_method, name, *args):
 class SqliteRegression(TestCase):
     def check_sqlite_regression(self, dataset, known_md5):
         """Check for regression for a particular dataset imported to sqlite"""
+        os.chdir(os.getcwd())
         os.system("rm output_database") #reinstalling changes checksum in sqlite
         os.system("retriever install sqlite %s -f output_database" % dataset)
-        os.system("echo '.dump' | sqlite3 output_database > output_file")
-        current_md5 = getmd5('output_file')
+        os.system("echo .dump | sqlite3 output_database > output_file")
+        current_md5 = getmd5(unixfileformat("output_file")) 
         assert current_md5 == known_md5
 
-
 class CSVRegression(TestCase):
     def check_csv_regression(self, dataset, known_md5):
         """Check for regression for a particular dataset imported to csv"""
+        os.chdir(os.getcwd())
```

Comment:
Is this necessary? It seems like it would just look up the current working directory and the set the current working directory to that value, which seems the same as not running this line in the first place. If we do need this line we should probably add a comment to it since it's difficult (for me at least) to understand what it's accomplishing.